packages/next-templates/src/app/globals.css

```diff
@@ -299,6 +311,21 @@ utrecht-alert
   justify-content: space-between;
 }
 
+.voorbeeld-container-info {
+  --toptask-inline-size-gaps-included: 752px;
+  padding-inline-end: var(--voorbeeld-space-inline-cat);
+  max-inline-size: calc(100% - var(--toptask-inline-size-gaps-included));
+}
+
+.example-toptask-header {
+  gap: var(--voorbeeld-space-inline-cat);
+}
+
+.example-toptask-header svg {
+  color: var(--voorbeeld-color-violet-800);
+  scale: 1.6;
+}
+
```

Comment:
De build is kapot, dus ik kan niet zien waarvoor dit gebruikt wordt. Als het voor een icon is, dan zou ik `<UtrechtIcon>` gebruiken als wrapper met de `icon-size` en `icon-color` variables.